### ISSUE TITLE
Any plan to reproduce the atari experiments as well?

### ISSUE THREAD

**Issue by hukz18** [nikhilbarhate99/min-decision-transformer#4]
Thank you for your simplified code! The original dt repo uses minigpt for the atari experiments as well, so a reproduction is (theoretically) possible. Is there any plan to include this part?

**Comment by nikhilbarhate99** [comment]
Hey,
Currently I do no have any plans for reproducing the atari experiments given the goal of this repo is to be simple and minimal implementation that can run on free resources on google colab.

You can try it out in a fork of this repo.

For reference, some of the high level things that will need to change are:
1. Dataloader and Data preprocessing / normalization
3. Adding an image encoder for atari input frames
4. Evaluation function
5. The hyperparameters
